src/components/Launcher.js

```diff
@@ -84,11 +85,13 @@ Launcher.propTypes = {
   messageList: PropTypes.arrayOf(PropTypes.object),
   mute: PropTypes.bool,
   showEmoji: PropTypes.bool,
+  typing: PropTypes.bool
 };
 
 Launcher.defaultProps = {
   newMessagesCount: 0,
-  showEmoji: true
-}
+  showEmoji: true,
+  typing: false
```

Comment:
Would we want this to default to `true`? I'm not sure..
If we default `true`:
- yay, fancy typing indicator everywhere

If we default `false`:
- maybe nice for users just trying to set up rcw who don't want to deal with setting up typing indicators too

cc @dharness for opinion

Reply:
I think the default has to be `false` - a default of `true` would display the typing indicator even if the user didn't set the prop.